vcluster/configure/vcluster-yaml/sync/from-host/ingress-classes.mdx

```diff
@@ -13,20 +13,84 @@ import EnableSwitch from '../../../../_partials/config/sync/fromHost/ingressClas
+              - nginx
+              - traefik
 ```
 
+The selector section uses the same syntax as the Kubernetes label selector, allowing you to filter which IngressClasses should be synced from the host based on their labels.
+Note that all the criteria must match for an IngressClass to be included in the sync.
+
+### How selector works
+
+`matchLabels` and `matchExpressions` are used to filter IngressClasses based on their labels.
+- matchLabels: This section allows you to specify exact label matches. For example, if you want to sync only IngressClasses with the label `environment: development`, you would specify it here.
+- matchExpressions: This section allows for more complex queries. For example, if you want to sync IngressClasses with the label `kubernetes.io/ingress.class` that are either `nginx` or `traefik`, you would specify it here using the `In` operator. Allowed operators include `In`, `NotIn`, `Exists`, and `DoesNotExist`.
+
+Refer to the [Kubernetes documentation on label selectors](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements) for more details on how to use label selectors effectively.
+
+When you enable the syncing of IngressClasses from the host cluster, the vCluster syncer looks for IngressClasses that match the specified selector criteria.
+
+The selector limits not only the IngressClasses that are synced from the host but also the sync of the Ingress resources created in the virtual cluster to the host.
+This ensures that only the relevant classes are available for use. So when trying to create an Ingress resource in the virtual cluster, it only allows the IngressClasses that match the specified selector criteria.
+
+The Ingress resource sync to host fails when using an IngressClass that does not match the selector. An event is recorded for the Ingress in the virtual cluster, indicating that the IngressClass is not available in the host.
+This should look like this:
+
+```shell
+vcluster-virtual-cluster-1:~$ kubectl describe ingress my-ingress
+Name:             my-ingress
+Namespace:        default
+Ingress Class:    nginx
+Events:
+  Type     Reason            Age   From                Message
+  ----     ------            ----  ----                -------
+  Warning  SyncWarning       10s   ingress-syncer      did not sync ingress "my-ingress" to host because it does not match the selector under 'sync.fromHost.ingressClasses.selector'
+```
+
+### Use cases
+- **Development Environments**: Sync specific IngressClasses that are used in development environments.
+- **Multi-Tenancy**: Allow different teams to use their own IngressClasses while still sharing the same host cluster resources.
+- **Security**: Limit the IngressClasses available in the virtual cluster to only those that are necessary.
+
+### Important notes
+- The synced IngressClasses is synced to the virtual cluster, allowing you to create Ingress resources that reference them.
```

Comment:
This sentence seems redundant. Also not particularly noteworthy. I would just remove this line.